### DISCUSSION TITLE
Why is the fuel prediction all weird?

### DISCUSSION THREAD

**Discussion by AirbusFlyer** [flybywiresim/aircraft#2328] [Q&A] [answered]
<img width="173" alt="Why" src="https://user-images.githubusercontent.com/57779477/101698424-4bd3a400-3a2e-11eb-96a3-05179b62ae80.PNG">
Does anyone know why this is happening or how I can fix it/if this is a bug?

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
Departure and arrival locations?

**Reply by AirbusFlyer** [reply]
24N-EGAC Maybe because it thinks the distance I am flying is out of the aircraft's capabilities? The flight plan I entered was 4002NM compared to the listed range of 3500NM. I did some testing and calculations and the actual range of the plane is between 5500-6100NM (I did this because I knew I was doing a long flight and wanted to see how far I could actually fly) depending on altitude. I also found higher altitude typically = higher fuel consumption and that the speed gain at higher altitudes is not worth the extra fuel. I found the optimal altitude to be around FL100-200 (If optimizing for best range and sacrificing speed).

**Reply by MapleNAV-TCASFAULT** [reply]
If its out of range its gonna give some weird values. Place an issue on it if this has not been mentioned yet.

**Reply by AirbusFlyer** [reply]
Ok thank you!

**Reply by Lucky38i** [reply]
Yeah unfortunately it only has coeffecients up to 3100NM, this is because thats the highest the performance data will go up to (or atleast what I have access to) so anything above that will produce skewed results.

Actually after, looking at the current dev you shouldn't have received any predictions at all. I'm not sure why this happened.

Second Edit!
I figured the issue, this should be fixed in #2251
